src/app/Main/Header/Export/Header-ExportContainer.component.tsx

```diff
@@ -20,10 +22,22 @@ import { generateExportLabels } from "./ExportServices/generateExportLabels.serv
 import { ApiResponse, callApiService } from "../../../Core/callApi.service";
 import { HeaderExportButtonComponent } from "./Header-ExportButton.component";
 import { ExportButtonLabelComponent } from "../../../Shared/Export-ButtonLabel.component";
+import { bfrPrimaryPalette } from "../../../Shared/Style/Style-MainTheme.component";
+
+const subheaderStyle = css`
+    width: 100%;
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    background-color: ${bfrPrimaryPalette[300]};
+    box-sizing: border-box;
+    box-shadow: 0 8px 6px -6px grey;
+`;

```

Comment:
Reject: See other comment. In general a container should not contain any styles. Styles belong to the view and layout components, the container is for handling app logic and putting together views and layouts.

Reply:
Done. Remove Style from Container